salt/modules/saltutil.py

```diff
@@ -623,6 +623,44 @@ def sync_thorium(saltenv=None, refresh=False, extmod_whitelist=None, extmod_blac
     return ret
 
 
+def sync_liquid(saltenv=None, refresh=False, extmod_whitelist=None, extmod_blacklist=None):
+    '''
+    .. versionadded:: Oxygen
```

Comment:
Note: I don't necessarily aim to get this into Oxygen. But if I can finish before the freeze, it may be nice to have.

Reply:
This won't make it into Oxygen as we've already branched after the feature freeze. So this will need to change to `Fluorine`.